benchmarks/bench_sparse_mla_sm120.py

```
@@ -0,0 +1,323 @@
+        print(f"{h:>10}  {k:>6}  {t:>11}  {lat_us:>10.1f}  {kvbw:>13.1f}  {tfl:>12.2f}")
+
+    print()
+    print("DSv4 prefill path (num_tokens > 64):")
+    print(header)
+    print("-" * len(header))
+    for h, k, t in prefill_configs:
+        lat_us, kvbw, tfl = bench_sparse_mla_sm120(h, k, t)
+        print(f"{h:>10}  {k:>6}  {t:>11}  {lat_us:>10.1f}  {kvbw:>13.1f}  {tfl:>12.2f}")
+
+    # DSv3.2 decode-dsv3_2: topk fixed at 2048, page_block_size=1, 656 B/token.
+    # Sweep is num_heads × num_tokens (with_sink off; sink is a per-head
```

Ruff (0.15.13)

[warning] Comment contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF003)